R/timer.R

```diff
@@ -96,3 +119,25 @@ scheduleTask <- function(millis, callback) {
     invisible(timerCallbacks$unschedule(id))
   }
 }
+
+#' Get a scheduler function for scheduling tasks. Give priority to the
+#' session scheduler, but if it doesn't exist, use the global one.
+#' @noRd
+defineScheduler <- function(session){
```

Comment:
Don't need this; can always use `session$scheduleTask` instead.

Reply:
Make this and the now function prefixed with a `.` and add both to the base session. Then get rid of these helpers and always use the version dangling off the session.

Add comments to the method definition in R6, too.